README.md

```diff
@@ -176,6 +144,97 @@ GTO the artifact file is committed to Git.
 
 </details>
 
+### Unassigning a stage
+
+Sometimes you need to mark an artifact version no longer ready for a specific
+consumer, and maybe signal a downstream system about this. You can use
+`gto untag` for that:
+
+```console
+$ gto untag awesome-model --version v0.0.1 --stage prod
```

Comment:
same as previous comment: how about this?

`gto untag awesome-model --stage prod --from-version v0.0.1 `